tests/phpt/smart_casts/05_inside_operators.php

```diff
@@ -0,0 +1,66 @@
+@ok
+<?php
+require_once 'kphp_tester_include.php';
+
+// these tests didn't fork until create_full_cfg was used
```

Comment:
I'm not sure I understand what this comment means. :)
Does it have a typo?

fork -> work?
until -> when?

```
these tests didn't work when create_full_cfg was used
```